benchmark/benchmark_runner.py

```diff
@@ -260,17 +263,48 @@ def run(
             rfc = RandomForestClassifier(
                 num_workers=self.args.gpu_workers, verbose=7, **params
             )
-            benchmark_string = "SparkCuml RandomForestClassifier training:"
+            benchmark_string = "SparkCuml RandomForestClassifier"
```

Comment:
Should probably change to "Spark RAPIDS ML" instead of "SparkCuml"

Reply:
Done. Thx